pkg/coscheduling/core/core.go

```diff
@@ -216,6 +216,7 @@ func (pgMgr *PodGroupManager) GetPodGroup(pod *corev1.Pod) (string, *v1alpha1.Po
 	return fmt.Sprintf("%v/%v", pod.Namespace, pgName), pg
 }
 
+// Calculate the number of bound pods. The Bound pods means Reserved/Assumed status in Scheduling Cycle, not the pods bonded to the nodes.
```

Comment:
How about this？
calculateBoundPods returns the count of pods that have occupied resources (including assumed and bound).

Reply:
I agree, I`ll change it.

Reply:
I hava updated the comment.